ui/ui-frontend/projects/archive-search/src/app/archive/archive-search/additional-actions-search/reclassification/reclassification.component.ts

```diff
@@ -211,6 +211,7 @@ export class ReclassificationComponent implements OnInit, OnDestroy {
       this.space +
       this.translateService.instant('RECLASSIFICATION.FIRST_STEP.CHILDS');
 
+    this.actions.find((action) => action.key === REPLACE).disabled = !this.archiveUnitAllunitup.length;
     this.actions.find((action) => action.key === PULL).disabled = !this.archiveUnitAllunitup.length;
```

Comment:
Tu pourrais rassembler les deux traitements en un seul. Quelque chose comme (non testé) :
```suggestion
    this.actions.filter((action) => action.key === REPLACE || action.key === PULL).map((action) => action.disabled = !this.archiveUnitAllunitup.length);
```

Mais c'est pas bloquant.

Reply:
@ebernard Done